src/olympia/reviewers/tests/test_utils.py

```diff
@@ -1098,7 +1098,7 @@ def test_nomination_to_public(self):
 
     def test_old_nomination_to_public_bonus_score(self):
         self.sign_file_mock.reset()
-        self.setup_data(amo.STATUS_NOMINATED, type=amo.ADDON_PLUGIN)
+        self.setup_data(amo.STATUS_NOMINATED, type=amo._ADDON_PLUGIN)
```

Comment:
lets change this to `ADDON_DICT` instead - there may be some fixes needed after the change though